kbncontent.go

```diff
@@ -137,6 +137,7 @@ func (v VisualizationDescriptor) HasFilters() (bool, error) {
 	queryPaths := []string{
 		"attributes.kibanaSavedObjectMeta.searchSourceJSON.query.query",
 		"attributes.state.query.query",
+		"embeddableConfig.attributes.mapStateJSON.query.query",
```

Comment:
Should you also add the path `embeddableConfig.attributes.mapStateJSON.filters`? 

Maps uses Kibana's unified search, which applies filtering from 2 locations. `query` captures the contents search bar while `filters` captures the contents of the filter bar. Both should be checked to know if a map panel applies a panel level filtering.

Reply:
I added it in https://github.com/elastic/kbncontent/pull/5/files#diff-922b6d6bcd00ced7f244a57230a1075d65e42bae8fe3e8c8ac18ce9b82d03d9cR154.

The structure for queries and filters are different, so there are two different checks, each one with its own list of paths.